Add-ons/UmbracoForms/Editor/Attaching-Workflows/index.md

```diff
@@ -1,60 +1,31 @@
 # Attaching workflows
-Submitting a form will result in the record data being stored in the database, if you wish to attach extra functionality to your form you can do so by assigning 1 or multiple workflows (like sending an email).
 
-## Adding a workflow
+In this article you can learn how to add extra functionality to your form by attaching **workflows**.
 
-### Navigate to the workflow section at the bottom of your form
-Select the *configure workflow* option in the bottom right corner.
+Workflows are a way of defining what happens when a form is submitted. This could be sending an email, or creating a content node.
 
-![Button](workflowbutton.png)
+Submitting a form will by default result in the record data being stored in the database. This can be configured in the Forms settings: [Store records](https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Editor/Creating-a-form/Form-Settings/#store-records-version-7).
```

Comment:
```suggestion
Submitting a form will by default result in the record data being stored in the database. This can be configured in the Forms settings: [Store records](../Creating-a-form/Form-Settings/#store-records-version-7).
```
We should use relative links 😉